build/src/components/05-gallery/_gallery-featherlight.scss

```diff
@@ -19,8 +19,7 @@
     width: 100%;
     height: 100%;
     max-height: 910px;
```

Comment:
This arbitrary max-height is not present in the default implementation of featherlight, is this really needed?